README.md

```diff
@@ -174,6 +174,12 @@ Use [Python index visualizer](./utils/visualizer/README.md) for your indexed tab
 
 Check [here](https://docs.delta.io/latest/releases.html) for **Delta Lake** and **Apache Spark** version compatibility.  
 
+## Format changes
+
+Version 1.0.0 introduced changes in the index metadata format which are
```

Comment:
`0.6.0` instead of `1.0.0`?

Reply:
Yep, need to merge this doc with the `QbeastFormat` instead of maintaining two separate files. 

Maybe we can use this second to state the problem with Replication. WDYT?